app/src/pages/trace/SpanAside.tsx

```diff
@@ -0,0 +1,148 @@
+`;
+
+/**
+ * A component that shows the details of a span that is supplementary to the main span details
+ * @returns
+ */
+export function SpanAside(props: { span: SpanAside_span$key }) {
+  const [data] = useRefetchableFragment<SpanAsideSpanQuery, SpanAside_span$key>(
+    graphql`
+      fragment SpanAside_span on Span
+      @refetchable(queryName: "SpanAsideSpanQuery") {
+        id
+        code: statusCode
+        startTime
+        endTime
+        tokenCountTotal
+        tokenCountPrompt
+        tokenCountCompletion
+        spanAnnotations {
+          id
+          name
+          label
+          annotatorKind
+          score
+        }
+      }
+    `,
+    props.span
+  );
+  const {
+    startTime,
+    endTime,
+    code,
+    tokenCountCompletion,
+    tokenCountPrompt,
+    tokenCountTotal,
+  } = data;
+  const startDate = useMemo(() => new Date(startTime), [startTime]);
+  const endDate = useMemo(
+    () => (endTime ? new Date(endTime) : null),
+    [endTime]
+  );
+  const latencyMs = useMemo(() => {
+    if (!endDate) return null;
+    return endDate.getTime() - startDate.getTime();
+  }, [endDate, startDate]);
+  const statusColor = useSpanStatusCodeColor(code);
+  const annotations = data.spanAnnotations;
+  const hasAnnotations = annotations.length > 0;
+  return (
+    <View
+      padding="size-200"
+      borderColor="dark"
+      backgroundColor="light"
+      borderLeftWidth="thin"
+      width="210px"
+      flex="none"
+      minHeight="100%"
+    >
+      <Flex direction="column" gap="size-200">
+        <LabeledValue label="Status">
+          <Flex direction="row" gap="size-50" alignItems="center">
+            <SpanStatusCodeIcon statusCode={code} />
+            <Text textSize="xlarge" color={statusColor}>
+              {code}
+            </Text>
+          </Flex>
+        </LabeledValue>
+        <LabeledValue label="Start Time">
+          <Text textSize="xlarge">{fullTimeFormatter(startDate)}</Text>
+        </LabeledValue>
+        {endDate && (
+          <LabeledValue label="End Time">
+            <Text textSize="xlarge">{fullTimeFormatter(endDate)}</Text>
+          </LabeledValue>
+        )}
+        <LabeledValue label="Latency">
+          <Text textSize="xlarge">
+            {typeof latencyMs === "number" ? (
+              <LatencyText latencyMs={latencyMs} textSize="xlarge" />
+            ) : (
+              "--"
+            )}
+          </Text>
+        </LabeledValue>
+        {tokenCountTotal ? (
+          <LabeledValue label="Total Tokens" key="tokens">
+            <TokenCount
+              tokenCountTotal={tokenCountTotal}
+              tokenCountPrompt={tokenCountPrompt ?? 0}
+              tokenCountCompletion={tokenCountCompletion ?? 0}
+              textSize="xlarge"
+            />
+          </LabeledValue>
+        ) : null}
+        {hasAnnotations && (
+          <LabeledValue label="Feedback">
+            <Flex direction="row" gap="size-50">
```

Comment:
is this flex necessary? it's only wrapping one component the`ul` which is it's own flexbox which controls the layout of it's children, seems unneeded

Reply:
Oh good catch